Payments/Operation/00_Operation.md

```diff
@@ -0,0 +1,25 @@
+# Operation
+
+In this part of the Payments documentation, you will find the following chapters:
+- [Monitor authorizations](./01_ManageAuthorizations.md)
+     - [Check authorization transaction](./01_ManageAuthorizations.md#check-an-authorization-transaction)   
```

Comment:
```suggestion
     - [Check authorization transaction](./01_ManageAuthorizations.md#check-authorization-transaction)   
```
Header in der Datei heißt Check authorization transaction, deswegen funktioniert der Link nicht (man landet in die Datei, aber nicht im richtigen Abschitt). Überleg es dir Link, wie du es nennen möchtest und passen den Link oder den Header dementsprechend. Bitte alle Links prüfen.

Reply:
Done